.travis.yml

```diff
@@ -1,10 +1,6 @@
 language: php
 
 php:
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
   - 7.1
   - 7.2
```

Comment:
We can add the `php-7.3` test in Travis CI build because this version is stable.

Reply:
hmm tests  in php 7.3  don't pass 😭

Reply:
The Travis CI build in `php-7.3` due to the failed assertions.

And it looks like the `BCMath` will be the decimal value result in every calculation.

Reply:
yep will search solution for this later

Reply:
I tried few solutions here and only one that is clean is to remove all trailing zeros in all methods. In math 1 eq 1.0 or 15.5 qe 15.50 etc so I don't see the problem cleaning all this and don't keep scale.

Reply:
@krowinski, just give you the message.

According to the official `bcmul` function [reference](https://www.php.net/manual/en/function.bcmul.php), the calculated result will be with trailing zeros since the PHP version is `7.3`.